command/plugin_info.go

```diff
@@ -22,14 +23,15 @@ func (c *PluginInfoCommand) Synopsis() string {
 
 func (c *PluginInfoCommand) Help() string {
 	helpText := `
-Usage: vault plugin info [options] NAME
+Usage: vault plugin info [options] NAME TYPE
```

Comment:
Commenting as a reminder to change it to the format we decide on.

Reply:
Thanks! Pushed!